### DISCUSSION TITLE
[Scam Alert] SynchroBit صرافی سینکروبیت

### DISCUSSION THREAD

**Discussion by saeedkousha** [Webamoozcom/warning-list#52] [Announcements]
<div dir="rtl">

**Project Name:** 
SynchroBit
 
**نام پروژه:** 
صرافی سینکروبیت

**نماد پروژه:**
SNB
**اتفاقی که افتاده است:**

SynchroBitcoin با نماد اختصاری SNB یک ارز دیجیتال یا شکلی از دارایی دیجیتال است که با ارزش بازار حدود ۲.۵۴ میلیون دلار، در رتبه ۱۴۰۶ بازار قرار داشته و سهم ۰.۰۰۰۱ درصدی از کل بازار را در اختیار دارد . هر واحد از SynchroBitcoin در این لحظه با قیمت ۰.۰۳۱۹۸۶ دلار، با احتساب نرخ تتر ۲۸,۲۹۰ تومان معادل ۹۰۴ تومان معامله می شود و حجم مبادلات روزانه آن ۳۸ دلار است. قیمت در ۲۴ ساعت اخیر -۱.۴۵% کاهش یافته است. بالاترین قیمت SynchroBitcoin در تاریخ ۰۵ شهریور ۱۳۹۹(26 August 2020) معادل ۰.۶۵ دلار بوده که همینک %۹۵.۰۹ پایین‌تر از آن زمان قرار دارد. تعداد واحدهای در گردش SynchroBitcoin ۷۹.۵۴ میلیون و تعداد کل واحدهای آن ۱ میلیارد خواهد بود. در حال حاضر فعال ترین صرافی که در آن SynchroBitcoin معامله می‌شود، صرافی Livecoin با سهم %۸۲.۸۱ از حجم معاملات روزانه است.

![188899ec-658e-461f-83f1-2e6e23848a2d](https://user-images.githubusercontent.com/91057696/135748265-3e7a81a0-a252-46d5-a71c-908cb3d90c5e.jpg)

نکته جالب در خصوص سینکروبیت و شخص آقای بابک بهبودی به عنوان مدیر عامل این هست که هیچ کدام از این دفاتر حقیقی نیستن و اطلاعات فیک میباشد . 

**قوانینی که نقض شده است:**

**نوع هشدار:**
هشدار ریسک بالا
**نوع کلاهبرداری:** 
شیدکوین 
پلت فرم تبادل
**وضعیت کلاهبرداری:**
در جریان

**نام کلاهبرداران:**
دکتر بابک بهبودی

https://instagram.com/drbehboudi?utm_medium=copy_link
ceo@synchrobit.io

**نام شرکت(ها):**
سینکروبیت

**آدرس وب سایت(ها):**
https://synchronium.io  
https://snbtoken.io

**برآورد مبلغ کلاهبرداری:**

**منابع:**
https://elmvaservat.com/%D9%87%D8%B4%D8%AF%D8%A7%D8%B1-%D9%85%D8%B1%D8%A7%D9%82%D8%A8-%DA%A9%D9%84%D8%A7%D9%87%D8%A8%D8%B1%D8%AF%D8%A7%D8%B1%DB%8C-%D9%87%D8%A7%DB%8C-%D8%AD%D9%88%D8%B2%D9%87-%D8%B1%D9%85%D8%B2%D8%A7%D8%B1/

https://baratnews.ir/iranian-blocchin-association/
**جمع بندی:**

**Comment by mrjalaliyeh** [comment]
مستندات را خواندم ولی قانع کننده نبود، توکنشون برای جذب سرمایه نیست و تا جایی که دیدم، برای بازاریابی از این توکن استفاده می‌کنند مثلا برای لینک دعوت یا مثلا ثبت نام و اولین خرید تراکنش در صرافی، با این تفاسیر نمیشه رو توکنشون مانور داد، در واقع صرافی غیرمتمرکز هستند و ایرادی در سایتشون ندیدم بجز اینکه اگر ۲ بار درخواست تایید هویتت قبول نشه باید چند دلار پرداخت کنی تا درخواستت بررسی بشه

**Reply by saeedkousha** [reply]
![188899ec-658e-461f-83f1-2e6e23848a2d](https://user-images.githubusercontent.com/91057696/135811015-bca90b68-16e2-4ae1-b05b-a60cb4fe4163.jpg)
 
تمامی اطلاعات دفاترشون فیک هستن و همین نکته بزرگ ترین دلیلی هست که باید بهشون شک کنین

**Reply by mrjalaliyeh** [reply]
از چه نظر فیک هستند؟ کدامیک از دفاتر را به صورت حضوری بررسی نموده‌اید؟

بنده پروفایل‌های شبکه‌های اجتماعی را بررسی کردم

این پروفایل داخلش نوشته شده که مدت 10 ماه مقام سی ای او سینکرونیوم رو داشته
https://www.linkedin.com/in/chuckhermans/
از طرفی طبق لینک زیر ایشون یک استاد دانشگاه با سابقه هست
https://www.researchgate.net/profile/Charles-Hermans

اینا ثابت میکنه که این شرکت یک فضای حرفه‌ای دارد و یک شرکت خیالی نیست

**Reply by Webamoozcom** [reply]
حسب اطلاع غیر ترکیه بقیه دفاتر چک شده اند و دفاتر واقعی نبوده اند.
